charts/fybrik/values.yaml

```diff
@@ -50,6 +50,24 @@ global:
   # zerolog verbosity level 
   # ref: https://github.com/rs/zerolog#leveled-logging
   loggingVerbosity: -1
+  # If deploying to OpenShift
+  openshift: false
+  # Pod Security Context
+  # ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
```

Comment:
please add a comment that this is the default setting for all pods, and can be overwritten by a specific pod settings
maybe use this link https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context

Reply:
Fixed. Thanks